src/locale/locales/de/messages.po

```diff
@@ -89,42 +85,41 @@ msgstr "Eine Inhaltswarnung hinzufügen"
 
 #: src/view/screens/ProfileList.tsx:744
 msgid "Add a user to this list"
-msgstr "Einen Benutzer zu dieser Liste hinzufügen"
+msgstr "Einen Nutzer zu dieser Liste hinzufügen"
 
-#: src/view/screens/Settings.tsx:355
-#: src/view/screens/Settings.tsx:364
+#: src/view/screens/Settings.tsx:355 src/view/screens/Settings.tsx:364
 msgid "Add account"
 msgstr "Konto hinzufügen"
 
 #: src/view/com/composer/photos/Gallery.tsx:119
 #: src/view/com/composer/photos/Gallery.tsx:180
 msgid "Add alt text"
-msgstr "Alt Text hinzufügen"
+msgstr "Alt-Text hinzufügen"
 
 #: src/view/com/modals/report/InputIssueDetails.tsx:41
 #: src/view/com/modals/report/Modal.tsx:191
 msgid "Add details"
-msgstr "Einzelheiten hinzufügen"
+msgstr "Details hinzufügen"
 
 #: src/view/com/modals/report/Modal.tsx:194
 msgid "Add details to report"
-msgstr "Einzelheiten zum Bericht hinzufügen"
+msgstr "Details zum Report hinzufügen"
 
 #: src/view/com/composer/Composer.tsx:442
 msgid "Add link card"
-msgstr "Link Karte hinzufügen"
+msgstr "Link-Karte hinzufügen"
```

Comment:
"Karte" (card) is not common in this context. Suggestion: "Link-Vorschau hinzufügen" (Vorschau = preview)